static/resources/js/register.js

```diff
@@ -39,9 +39,10 @@ jQuery(function(){
             var isAdmin = $("#isAdmin").val();
             
 			$.ajax({
```

Comment:
Try to user AjaxUtil as for consistency. It's a wrapper of primitive ajax call, you can refer to other JS files for the usage.